lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,137 @@
+unzip ibm-eventstore-developer-client.1.1.1.zip
+    ```
+    3. Install the library on your local repository:
+    ```
+mvn install:install-file -Dfile=ibm-eventstore-developer-client.1.1.1/ibm-event.jar -Dversion=1.1.1 -DgroupId=com.ibm.eventstore -DartifactId=ibm-event -Dpackaging=jar
+    ```
+
+
+## Start the Lagom sample application
+
+In the command line shell where you downloaded the Lagom service, from the `lagom-eventstore-example` directory, start the Lagom development environment by running:
+
+```
+mvn lagom:runAll
+```
+
+You should see some console output, including these lines:
+
+```
+...
+[INFO] Service gateway is running at http://localhost:9000
+...
+[INFO] (Service started, press enter to stop and go back to the console...)
+```
+
+These messages indicate that the service has started correctly.
+
+
+## Generate some traffic on the Lagom service
+
+From a new terminal, you should now generate some traffic on the Lagom service. This is a dummy service with a very simple `GET` operation where users may say hello to other users. Each time a request to say hello to `user123` an event is emitted and stored on IBM Project EventStore. For efficiency, events are not stored immediately, instead they are batched and stored every 5th greeting a user receives.
+
+To send some greetings you should:
+
+1.  Open a new terminal
+2.  `curl http://localhost:9000/api/hello/Alice` 
+3.  (repeat the previous step several times)
+
+You may also want to send greetings to other users:
+
+1.  `curl http://localhost:9000/api/hello/Bob` 
+2.  (repeat the previous step several times)
+3.  `curl http://localhost:9000/api/hello/Steve` 
+4.  (repeat the previous step several times)
+
+
+Here you should use a traffic generation tool like [JMeter](http://jmeter.apache.org/), [Gatling](http://gatling.io/), [ab](https://httpd.apache.org/docs/2.4/programs/ab.html) or [wrk](https://github.com/wg/wrk) to generate massive amounts of requests.
+
+For example:
+
+```
+wrk http://localhost:9000/api/hello/Janine
+wrk http://localhost:9000/api/hello/Albert
+wrk http://localhost:9000/api/hello/Billy
+wrk http://localhost:9000/api/hello/Samantha
+```
+
+## Analyse the traffic using IBM Project EventStore Notebooks
+
+Once you have generated some data you should use the UI provided by IBM Project EventStore to analyse it:
+
+1.  Bring the UI of IBM Project EventStore to the foreground.
+2.  On the Top left corner, click on the menu Icon and select `My Notebooks`.
+3.  Click the `(+) Add notebooks` action in the top right section of the UI.
+4.  Select `From File` and fill the form fields `Name` and `Description`. In the `Notebook File` field, select the file `/lagom-ibm-integration-examples/lagom-eventstore-example/resources/lagom-event-store-example-greetings.ipynb` from your repository and click the `Create Notebook` button.
+5.  Once opened, use the `Run Cell` button (![](docs/imgs/run-cells.png)) to step forward executing each piece of your Notebook.
```

Comment:
It looks like the notebook needs to be edited in some places by the user to work correctly:

- The endpoint needs to be set by the user to use their public IP (localhost does not work here)
- There is a case mismatch between the default "GreetingsDB" in the notebook and "greetingsDB" in the Lagom config (this we should just fix)